debug_toolbar_line_profiler/panel.py

```diff
@@ -165,7 +168,9 @@ def process_view(self, request, view_func, view_args, view_kwargs):
         args = (request,) + view_args
         self.line_profiler = LineProfiler()
         self._unwrap_closure_and_profile(view_func)
-        if view_func.func_globals['__name__'] == 'django.views.generic.base':
+        view_func_name = view_func.__globals__['__name__'] if PY3 else \
+            view_func.func_globals['__name__'] 
+        if view_func_name == 'django.views.generic.base':
             for cell in view_func.func_closure:
```

Comment:
The `func_closure` is also not available in Python 3. Here should also be something like
```
            for cell in view_func.__closure__ if PY3 else \
                    view_func.func_closure:
```